src/env/child/mod.rs

```diff
@@ -150,9 +149,22 @@ impl<'a> ChildRuntime<'a> {
 
         Ok(Some(len.into()))
     }
+
+    #[cfg(feature = "debug")]
+    fn ext_print(&self, args: RuntimeArgs) -> ExtResult {
+        let memory = self.memory();
+
+        let ptr: u32 = args.nth(0);
+        let len: u32 = args.nth(1);
+
+        let bytes = memory.get(ptr, len as usize).unwrap();
+
+        self.root().print(&bytes);
+
+        Ok(None)
+    }
 }
 
-#[derive(Debug)]
```

Comment:
Might want to look at the [derive_more](https://crates.io/crates/derive_more) crate. It lets you skip fields when deriving debug.